nomenclature/testing.py

```diff
@@ -27,7 +27,7 @@ def assert_valid_yaml(path: Path):
         )
 
 
-def assert_valid_structure(path: Path):
+def assert_valid_structure(path: Path, mappings, definitions):
```

Comment:
Please make these arguments optional (with current usage as default) so that it is possible to use the function directly (without the CLI) as before.